dscript/models/interaction.py

```diff
@@ -241,7 +268,17 @@ def map_predict(
             phat = self.activation(phat).squeeze()
         return C, phat
 
-    def predict(self, z0, z1, embed_foldseek=False, f0=None, f1=None):
+    def predict(
```

Comment:
We should think about packaging inputs better-- maybe [z0, f0, b0] all get concatenated into a single tensor, and the arguments describe how to split? I'm not sure, but passing around this many parameters gets really quickly unwieldy and is likely to lead to bugs when we forget to update one function.